rpc/types_broadcast_transaction.go

```diff
@@ -82,15 +82,8 @@ func (tx BroadcastDeclareTxnV1) GetContractClass() interface{} {
 }
 
 type BroadcastDeclareTxnV2 struct {
-	Type TransactionType `json:"type"`
-	// SenderAddress the address of the account contract sending the declaration transaction
-	SenderAddress     *felt.Felt         `json:"sender_address"`
-	CompiledClassHash *felt.Felt         `json:"compiled_class_hash"`
-	MaxFee            *felt.Felt         `json:"max_fee"`
-	Version           TransactionVersion `json:"version"`
-	Signature         []*felt.Felt       `json:"signature"`
-	Nonce             *felt.Felt         `json:"nonce"`
-	ContractClass     ContractClass      `json:"contract_class"`
+	DeclareTxnV2
```

Comment:
Unfortunately, we can't do that because the `BroadcastDeclareTxnV2` doesn't have the 'ClassHash' field required by the `DeclareTxnV2` tx. Let's move it back.
Ref: https://github.com/starkware-libs/starknet-specs/blob/e5ce8c6f99672d9ddf85a8e1d6ef8e87169d6e8d/api/starknet_api_openrpc.json#L2068

Reply:
Done